tests/pdk/finfet_pdk/test_benchmark_place.py

```diff
@@ -0,0 +1,134 @@
+import time
+import json
+import pytest
+import shutil
+from .utils import build_example, run_example
+from . import circuits
+from align.pnr.hpwl import gen_netlist, calculate_HPWL_from_placement_verilog_d
+from align.pnr.render_placement import standalone_overlap_checker
+
+
+cleanup = True
+
+# PARAMS = [
+#         ("tamu_sp", ["--router_mode", "no_op"]),
+#         ("umn_ilp", ["--router_mode", "no_op", "--select_in_ILP"])
+# ]
+PARAMS = [
+        ("tamu_sp", [""]),
+        ("umn_ilp", ["--select_in_ILP"])
```

Comment:
is this test configuration no longer correct?

Reply:
Fixed it in the recent checkin.